.changeset/sweet-cows-burn.md

```diff
@@ -0,0 +1,6 @@
+---
+"@logto/core-kit": patch
```

Comment:
should be "minor"? new feature added

Reply:
not sure actually. since a new claim can be a feature, or can be just a patch, both sound reasonable?